packages/toolpad-app/jest.config.js

```diff
@@ -7,6 +7,18 @@ const transpiledModules = [
   'data-uri-to-buffer',
   'fetch-blob',
   'formdata-polyfill',
+  'd3-scale',
+  'd3-array',
+  'internmap',
+  'd3-interpolate',
+  'd3-color',
+  'd3-format',
+  'd3-time',
+  'd3-time-format',
+  'd3-shape',
+  'd3-path',
+  '@mui/x-charts/esm',
```

Comment:
that path is not exported from `@mui/x-charts`
```suggestion
  '@mui/x-charts',
```

Reply:
`@mui/x-charts` wasn't working to stop jest from showing an error, but `@mui/x-charts/esm` did... Maybe it's possible to use a glob or something though, I can try that.

Reply:
We don't allow imports from the esm folder: https://mui.com/material-ui/guides/minimizing-bundle-size/#how-to-use-custom-bundles.

Reply:
@apedroferreira just `'@mui/x-charts',` seems to work for me.

I'm going to try a migration to `vitest`, I've just about had it hunting down ESM issues with vite.

Reply:
=> https://github.com/mui/mui-toolpad/pull/2709

This took a fraction of the time I've spent fighting jest about ESM imports. Had to configure nothing at all, just works out of the box.